File: src/main/java/org/team1/nbe1_3_team01/domain/calendar/application/response/ScheduleIdResponse.java
package org.team1.nbe1_3_team01.domain.calendar.application.response;

import lombok.Builder;

@Builder
public record ScheduleIdResponse(
        Long scheduleId
) {

    public static ScheduleIdResponse from(Long scheduleId) {
        return ScheduleIdResponse.builder()
                .scheduleId(scheduleId)
                .build();
    }
}

File: src/main/java/org/team1/nbe1_3_team01/domain/calendar/application/response/ScheduleIdResponse.kt
package org.team1.nbe1_3_team01.domain.calendar.application.response

data class ScheduleIdResponse(
    val scheduleId: Long
) {
    companion object {
        fun from(scheduleId: Long): ScheduleIdResponse {
            return ScheduleIdResponse(
                scheduleId = scheduleId
            )
        }
    }
}
